model/src/main/java/org/projectnessie/api/params/CommitLogParams.java

```diff
@@ -119,6 +116,14 @@ public String filter() {
     return filter;
   }
 
+  public FetchOption fetchOption() {
+    return fetchOption;
+  }
+
+  public boolean isFetchAll() {
```

Comment:
Not sure about adding `isFetchAll()` to the `*Params` classes. It's duplicated in `GetReferenceParams` + `ReferencesParams`.

WDYT about making `fetchOption` non-null and annotate with `@DefaultValue("MINIMAL")` so it cannot become `null` and adding a method `isAll()` to `FetchOption`?

Reply:
we could probably do that, but we would then have to make sure that `fetchOption` is never null and validate that in the `validate()` methods of the *Param classes and also remove the `empty()` method from those.

Reply:
My concern is that it's a helper method for a particular attribute's value.
If there are more values in `FetchOption`, it would look weird to me to add more `isFetchFoo` methods.

Reply:
I moved `isFetchAll` into the enum itself. I think that makes things easier and we don't have repetitive code.